techniques/systemSettings/misc/clockConfiguration/3.0/tz_vars.st

```diff
@@ -0,0 +1,932 @@
+#  Copyright (c) 2015 Janos Mattyasovszky
+# 
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+# 
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+# 
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# 
+
+bundle agent tz_variables
+{
+  vars:
+
+    ## The values for some deprecated fields we plan to phase out in future versions
```

Comment:
These fields below are required to have the technique's behavior stay the same on an unchanged input from the drop-down menu, still using the Countries instead of the new City-List.
Could be removed in technique version 4.0?